README.md

```diff
@@ -1,2 +1,30 @@
 # liveDVR
 DVR implementation built on top of live streams
+
+# Installation #
+
+## Pre-requisites ##
+
+* Make sure node is installed
+* Install npm
+* Install nvm (curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.2/install.sh | bash)
+* Clone repository
+* C++ compiler
```

Comment:
Note that GCC >= 4.7 is required. This is due to the use of --stg=c++11 in node_addons/FormatConverter/binding.gyp.